sites/public/lib/helpers.tsx

```diff
@@ -40,6 +40,13 @@ const getListingCardSubtitle = (address: Address) => {
   return address ? `${street}, ${city} ${state}, ${zipCode}` : null
 }
 
+export const getImageTagFromListing = (listing: Listing) => {
```

Comment:
I think this works for now, but probably not ideal long term. Can we add a TODO to either delete the communityType or change the label?

Reply:
Done.